tests/components/streams_bootstrap/test_producer_app.py

```diff
@@ -60,11 +60,49 @@ def producer_app(self) -> ProducerApp:
                 },
             },
         )
+        assert producer.is_cron_job is False
+        return producer
+
+    @pytest.fixture()
+    def producer_app_cron_job(self) -> ProducerApp:
+        producer = ProducerApp(
```

Comment:
Should we use model_validate? :) 
```suggestion
        producer = ProducerApp.model_validate(
```

Reply:
definitely!